packages/kubernetes-1.22/kubelet-exec-start-conf

```diff
@@ -22,6 +22,9 @@ ExecStart=/usr/bin/kubelet \
     --image-credential-provider-bin-dir /usr/libexec/kubernetes/kubelet/plugins \
     --image-credential-provider-config /etc/kubernetes/kubelet/credential-provider-config.yaml \
 {{/if}}
+{{/if}}
+{{#if settings.kubernetes.hostname-override}}
+    --hostname-override {{settings.kubernetes.hostname-override}} \
```

Comment:
With kubelet on `standalone-mode`, the flag is just ignored i think since it doesn't need to sync with the kube API server:
```
# /etc/systemd/system/kubelet.service.d/exec-start.conf
[Service]
ExecStart=
ExecStart=/usr/bin/kubelet \
    --cloud-provider "" \
    --config /etc/kubernetes/kubelet/config \
    --container-runtime=remote \
    --container-runtime-endpoint=unix:///run/containerd/containerd.sock \
    --containerd=/run/containerd/containerd.sock \
    --root-dir /var/lib/kubelet \
    --cert-dir /var/lib/kubelet/pki \
    --hostname-override i-02164391bf35e4540.us-west-2.compute.internal \
    --node-ip ${NODE_IP} \
    --node-labels "${NODE_LABELS}" \
    --register-with-taints "${NODE_TAINTS}" \
    --pod-infra-container-image ${POD_INFRA_CONTAINER_IMAGE}


bash-5.1# cat /proc/sys/kernel/hostname
i-02164391bf35e4540.etung.test

bash-5.1# systemctl status kubelet     
● kubelet.service - Kubelet
     Loaded: loaded (/x86_64-bottlerocket-linux-gnu/sys-root/usr/lib/systemd/system/kubelet.service; enabled; vendor preset: enabled)
    Drop-In: /x86_64-bottlerocket-linux-gnu/sys-root/usr/lib/systemd/system/kubelet.service.d
             └─dockershim-symlink.conf
             /etc/systemd/system/kubelet.service.d
             └─exec-start.conf
             /x86_64-bottlerocket-linux-gnu/sys-root/usr/lib/systemd/system/kubelet.service.d
             └─load-ipvs-modules.conf, make-kubelet-dirs.conf, prestart-pull-pause-ctr.conf
     Active: active (running) since Wed 2023-04-19 21:45:35 UTC; 4min 58s ago
```

kubelet logs
```
Apr 19 21:45:34 i-02164391bf35e4540.etung.test kubelet[1192]: I0419 21:45:34.932401    1192 server.go:575] "Standalone mode, no API client"
Apr 19 21:45:34 i-02164391bf35e4540.etung.test kubelet[1192]: I0419 21:45:34.967964    1192 server.go:463] "No api server defined - no events will be sent to API server"
...
Apr 19 21:45:35 i-02164391bf35e4540.etung.test kubelet[1192]: I0419 21:45:35.065994    1192 state_mem.go:36] "Initialized new in-memory state store"
Apr 19 21:45:35 i-02164391bf35e4540.etung.test kubelet[1192]: I0419 21:45:35.087163    1192 kubelet.go:404] "Kubelet is running in standalone mode, will skip API server sync"
Apr 19 21:45:35 i-02164391bf35e4540.etung.test kubelet[1192]: I0419 21:45:35.087210    1192 kubelet.go:286] "Adding static pod path" path="/etc/kubernetes/static-pods/"
Apr 19 21:45:35 i-02164391bf35e4540.etung.test kubelet[1192]: I0419 21:45:35.089769    1192 kuberuntime_manager.go:244] "Container runtime initialized" containerRuntime="containerd" version="1.6.19+bottlerocket" apiVersion="v1"
Apr 19 21:45:35 i-02164391bf35e4540.etung.test kubelet[1192]: I0419 21:45:35.091502    1192 volume_host.go:75] "KubeClient is nil. Skip initialization of CSIDriverLister"
Apr 19 21:45:35 i-02164391bf35e4540.etung.test kubelet[1192]: W0419 21:45:35.092416    1192 probe.go:268] Flexvolume plugin directory at /var/lib/kubelet/plugins/volume/exec does not exist. Recreating.
Apr 19 21:45:35 i-02164391bf35e4540.etung.test kubelet[1192]: W0419 21:45:35.094222    1192 csi_plugin.go:189] kubernetes.io/csi: kubeclient not set, assuming standalone kubelet
Apr 19 21:45:35 i-02164391bf35e4540.etung.test kubelet[1192]: W0419 21:45:35.094240    1192 csi_plugin.go:266] Skipping CSINode initialization, kubelet running in standalone mode
Apr 19 21:45:35 i-02164391bf35e4540.etung.test kubelet[1192]: I0419 21:45:35.094427    1192 server.go:1186] "Started kubelet"
Apr 19 21:45:35 i-02164391bf35e4540.etung.test kubelet[1192]: I0419 21:45:35.094516    1192 kubelet.go:1502] "No API server defined - no node status update will be sent"
Apr 19 21:45:35 i-02164391bf35e4540.etung.test systemd[1]: Started Kubelet.




```